README.md

```diff
@@ -1,2 +1,24 @@
 # git-essentials
-This repository will walk through some basic of git and how to get started with it.
+Hello All, Here I listed few things About Git!
+
+What Is Git?
+* Git is a version control system for tracking changes in computer files.
+* It helps in coordinating work amongst several people in a project and tracks progress over time. Unlike the centralized version control system, Git branches can be easily merged. A new branch is created every time a developer wants to start working on something. 
+* This ensures that the master branch always has a production-quality code.
+* Git is a distributed version control system, so here, every developer gets their local repository with full commit history. 
+* The commit history makes Git fast, as now a network connection is not needed to create commits or perform diffs between commits.
```

Comment:
Change from `*` to `-` rest all looks good!

Reply:
Hello Thank you for reviewing
Kindly check the latest commit!

Reply:
LGTM! Thanks.